b2sdk/file_version.py

```diff
@@ -257,60 +257,3 @@ def __init__(self, file_id, file_name):
     def as_dict(self):
         """ represents the object as a dict which looks almost exactly like the raw api output for delete_file_version """
         return {'action': 'delete', 'fileId': self.file_id, 'fileName': self.file_name}
-
-
-class FileIdNameAndRetention(object):
```

Comment:
without this how will the user know whether the file was processed just before or just after midnight, to determine whether the file will stay governed until the 5th or until the 6th of the month?

Reply:
well, the user requests `retainUntilTimestamp` and not `retain for 7 days`, so all the data the server responds with is identical to what the user provides. My initial idea was to provide these, but @mzukowski-reef thinks it's a bad idea.

Reply:
correct!